### DISCUSSION TITLE
How Can I Quiet CLI Dumps

### DISCUSSION THREAD

**Discussion by silkquasar** [prisma/prisma#15824] [Q&A] [answered]
I began using Prisma today.  My terminal is filled with every single Prisma action, inserts, queries, selects, etc. What is the process to shut off all the info Prisma is sending to the terminal?

**Comment by nurul3101** [comment, marked as answer]
Hello @silkquasar 👋 
It seems that you would have logging enabled, could you try removing the log parameters?
While instantiating the Prisma Client, it should be something like this:
```js
const prisma = new PrismaClient({
  log: ['query', 'info', 'warn', 'error'],
})
```
You can change it to:
```js
const prisma = new PrismaClient()
```

This would disable logging.

[Logging Reference](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/logging)

**Reply by silkquasar** [reply]
awesome!! This is exactly what I need.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
